README.md

```diff
@@ -673,6 +675,14 @@ If you do not want to change the git configuration, you can also use:
 $ git push -u <remote> HEAD
 ```
 
+The following command will push all branches that exist locally and remotely with the same name:
+
+```sh
+$ git push
+```
+
+This command is usually appropriate when a developer pushes to his own public repository, but may be confusing if not dangerous when using a shared repository.
+
```

Comment:
In line with my previous comment, this command is not "dangerous" anymore with Git >= 2.0 ... unless we use `git config push.default current` ! :)

For this reason and those I mentioned in my above comments, I would like to suggest not to include at all the text
> The following command will always push to a remote branch named origin, even if there are other remote branches with different names:
> 
> ```sh
> $ git config push.default current
> $ git push -u
> ```

and just mention `git config push.default upstream` and/or give a link to the [doc](https://git-scm.com/docs/git-config#git-config-pushdefault)

Indeed it seems `current` has really no advantage over `upstream`: it is more "dangerous", we cannot choose the remote we want to use, and it does not shorten much the command to type:
compare `git push -u` (with `git config push.default current`)
and `git push -u origin HEAD` (with the default config or with `git config push.default upstream`)

Finally, if in some context one is not very confident about some push, one can always do `git push -n` ;)

Reply:
Hey @erikmd, thank you for pointing that out. I am not that familiar with ```upstream``` , but from your comment I understand that the command ```git push -u``` will be replaced by ```git push -u origin HEAD``` when I use ```upstream``` instead of ```current```, right? .

Reply:
Exactly.

Reply:
Alright, I'll add that then